tidb-binlog/pump_client/selector.go

```diff
@@ -132,7 +132,7 @@ func (r *RangeSelector) SetPumps(pumps []*PumpStatus) {
 }
 
 // Select implement PumpSelector.Select.
-func (r *RangeSelector) Select(binlog *pb.Binlog, retryTime int) *PumpStatus {
+func (r *RangeSelector) Select(binlog *pb.Binlog, _ int) *PumpStatus {
```

Comment:
How about removing the unused parameter?

Reply:
removed

Reply:
It cannot be removed. it is to implement this interface and this interface is important.